website/docs/d/is_subnets.html.markdown

```diff
@@ -34,15 +34,15 @@ resource "ibm_is_subnet" "example" {
   resource_group  = data.ibm_resource_group.example.id
 }
 
-data "ibm_is_subnets" "example_resource_group" {
+data "ibm_is_subnets" "example" {
```

Comment:
For all the examples:
Can we check for redundancy. 
example -> example1 or similar ...